pages/common/views/components/diff.jsx

```diff
@@ -10,7 +10,7 @@ const getValue = (key, { format } = {}) => {
 const Diff = ({
   diff,
   formatters = {},
-  comparator = (...args) => !(isEqual(...args)),
+  comparator = (...args) => !(isEqual(args[1], args[2])),
```

Comment:
`(a, b) => !isEqual(a, b)`

Not _everything_ needs a spread operator.

Reply:
FINE